cpp/src/gandiva/greedy_dual_size_cache.h

```diff
@@ -17,6 +17,8 @@
 
 #pragma once
 
+#include <llvm/ADT/SmallString.h>
```

Comment:
this is not required

Reply:
Ok, I've used the "Optmize imports" from Clion and it did not remove it before, so I removed it.